app/scripts/app-init.js

```diff
@@ -184,4 +184,12 @@ const registerInPageContentScript = async () => {
   }
 };
 
+chrome.runtime.onInstalled.addListener(function (details) {
+  if (details.reason === 'install') {
+    chrome.storage.session.set({ isFirstTimeInstall: true });
+  } else if (details.reason === 'update') {
+    chrome.storage.session.set({ isFirstTimeInstall: false });
```

Comment:
Nit: can we extract this to util ?

Reply:
I don't think there is much benefit as we can't import a file within app-init.js, and this will only be used one place within app-init.js

Reply:
makes sense